addons/danger/functions/fnc_suppress.sqf

```diff
@@ -36,7 +36,7 @@ if (!_override) then {
 
     _pos = AGLtoASL (_unit getHideFrom (_unit findNearestEnemy _unit));
     private _vis = lineIntersectsSurfaces [eyePos _unit, _pos, _unit, vehicle _unit, true, 2];
-    if (count _vis > 1) then {_pos = (_vis select 0) select 0;};
+    if !(_vis isEqualTo []) then {_pos = (_vis select 0) select 0;};
```

Comment:
Should final argument in line 38 also be 1 instead of two then.  No need to return two positions. ATM I cannot recall why I set it up in that manner.